README.md

```diff
@@ -4,6 +4,18 @@
 
 Python library (and some tools) for converting [ROS bagfiles](http://wiki.ros.org/rosbag) to [Pandas dataframes](https://pandas.pydata.org/).
 
+## Performance 
+
+This fork is twice as fast as the original version and four times more memory efficient. A real-time progress bar is provied for you as well.
```

Comment:
provied -- > provided

Reply:
I would remove this later since this PR will be merged eventually. We don't need this sort of benchmarking anymore.